app/experimenter/nimbus-ui/src/components/App/index.test.tsx

```diff
@@ -12,19 +12,14 @@ import {
 import App from ".";
 
 describe("App", () => {
-  it("renders as expected", () => {
-    render(<Subject />);
-    expect(screen.getByTestId("app")).toBeInTheDocument();
-  });
-
   it("routes to PageHome page", () => {
     render(<Subject path="/" />);
     expect(screen.getByTestId("PageHome")).toBeInTheDocument();
   });
 
-  it("routes to PageExperimentNew page", () => {
```

Comment:
Renamed this component per a slack convo toward simplifying some of these names with redundant "Experiment" in them.